modules/setting/lfs.go

```diff
@@ -13,6 +13,7 @@ import (
 // LFS represents the configuration for Git LFS
 var LFS = struct {
 	StartServer    bool          `ini:"LFS_START_SERVER"`
+	AllowPureSSH   bool          `ini:"LFS_ALLOW_PURE_SSH"`
```

Comment:
https://gitea.com/gitea/docs/pulls/49

Reply:
Should it be added to the `app.example.ini`, considering it's default false anyway?

Reply:
To be clear, the reason I'm asking the above is because the semantics would be different.
When later the key is set to default-true in code, existing installations would get it enabled automatically, as it's set up now. If the default ini sets it explicitly, they won't get it enabled until the admin knows/decides to do so.